src/Aquarius.Client/TimeSeries/Client/ServiceModels/Provisioning.cs

```diff
@@ -3384,6 +3426,12 @@ public class PostReflectedTimeSeries
         ///</summary>
         [ApiMember(DataType="string", Description="ISO 8601 Duration Format", Format="duration", IsRequired=true)]
         public Duration GapTolerance { get; set; }
+
+        ///<summary>
+        ///Property Bag
+        ///</summary>
+        [ApiMember(Description="Property Bag")]
+        public string PropertyBag { get; set; }
```

Comment:
Do the mappers for SamplesConnector need to be updated to include this property bag? It's probably fine and not used by Samples?

Reply:
SamplesConnector uses the AQUARIUS SDK so can't be updated until this is merged and an updated nuget package is available. This should be a non-breaking change as it is a new, optional parameter so shouldn't matter if it was not updated, just means the property can't be set.